HttpRequest.cpp

```diff
@@ -1,120 +0,0 @@
-#include "HttpRequest.h"
-#include <vector>

```

Comment:
yea good point, i figured it wouldn't be a prob since its all in the commit history. However this file is just a file renaming anyway.